stata_kernel/kernel.py

```diff
@@ -26,8 +26,10 @@ def __init__(self, *args, **kwargs):
 
         # Can't name this `self.config`. Conflicts with a Jupyter attribute
         self.conf = Config()
-        self.magics = StataMagics()
+        self.graphs = {}
+        self.graph_formats = ["svg", "png", "pdf", "tif"]
```

Comment:
Unless you're also changing `send_image`, this should be just `svg` and `png`. `pdf` support doesn't really exist... in Jupyter Notebook the user is presented with a link to open the PDF in a new tab. I don't see any reason why `tif` would improve on either `svg` or `png`; it's like 1-2 orders of magnitude larger too.

Do you use `self.graphs` again?

Reply:
I thought `self.graphs` was from master. I don't use it. I'll take out `pdf` and `tif'

Reply:
`self.graphs` was used a long time ago when I was trying to keep a manual record of what graphs were exported. I recently deleted it as I had forgotten about it.